rootToH5converter/Event.py

```diff
@@ -116,16 +133,19 @@ def fill_jets(self, use_fat_jets=False):
                       n_neutral=self.data_processor.get_value_from_tree(prefix+"Jet_nNeutral", self.i_event, i_jet),
                       ch_hef=self.data_processor.get_value_from_tree(prefix+"Jet_chHEF", self.i_event, i_jet),
                       ne_hef=self.data_processor.get_value_from_tree(prefix+"Jet_neHEF", self.i_event, i_jet))
-            
+
             # check if tree contains links between tracks and jets
             jet_index = -1
             track_jet_index = None
-            if self.data_processor.get_value_from_tree("Track_jet_index") is not None:
-                track_jet_index = self.data_processor.get_value_from_tree("Track_jet_index", self.i_event)
+            track_cand_index = None
+            if self.data_processor.get_value_from_tree("Track_jet_index_"+AKx) is not None:
+                track_jet_index = self.data_processor.get_value_from_tree("Track_jet_index_"+AKx, self.i_event)
```

Comment:
Do we need these checks here? I guess data_processor.get_value_from_tree(...) will just return None if it cannot find the value, so we anyway and up with None in such case, or value otherwise.

Reply:
We need it to know which input file it is (102X or 106X).
If it's Delphes, ` track_jet_index` is set to sth in the 1st condition, but will be over-written with `None` if the other condition is not there.

Reply:
Yes maybe you're right, I'll test it. I just kept the way it was done before.

Reply:
The if is not for line 142, but line 143, which needs to be in the `if`!

Reply:
Oh, I see! How about something like this (we get rid of two lines and a long "if":

```python
track_jet_index = self.data_processor.get_value_from_tree("Track_jet_index_" + AKx, self.i_event)
jet_index = -1 if track_jet_index is None else i_jet
```

Reply:
Yes that works, I'll change that